Bots/DotNet/Skills/CodeFirst/EchoSkillBot/EchoSkillBot.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.11.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.13.0-daily.20210308.220944.384fea4" />
```

Comment:
Why hard coding this version?

Reply:
This one is preexisting and I added it to use the bits that fix a bug in dialogmanager, this should go away when we update to 4.12 as part of #321